apps/server/src/utilities/file.service.ts

```diff
@@ -21,7 +21,9 @@ export class FileService {
     async getFileCount(directory: PathLike): Promise<number> {
         try {
             this.logger.debug(`Retrieving number of files in folder: ${directory}`);
-            const retrievedDir = await readdir(`${process.cwd()}/${directory}`);
+            const dir = join(process.cwd(), <string>directory);
```

Comment:
**suggestion:** let's move all of that to use `string` as the argument and not `PathLike`. I see no circumstance in which we will use a `Buffer` as a parameter to this function :smile: